### PR TITLE
Reformat code

### PR DESCRIPTION
**Pull request by AndriySvyryd**
Will push by EOD if no reviews

### CODE REVIEW

File: src/EFCore.Relational/Migrations/Internal/MigrationsModelDiffer.cs [hunk 1]
```diff
@@ -1221,7 +1222,8 @@ protected virtual IEnumerable<MigrationOperation> Remove([NotNull] ISequence sou
         protected virtual string[] GetColumns([NotNull] IEnumerable<IProperty> properties)
             => properties.Select(p => p.Relational().ColumnName).ToArray();
 
-        /// <summary>.Relational(
+        /// <summary>
+        ///     .Relational(
```
Comment (bricelam) [inline src/EFCore.Relational/Migrations/Internal/MigrationsModelDiffer.cs]: Remove

File: src/EFCore.Relational/Storage/RelationalConnection.cs [hunk 2]
```diff
@@ -267,7 +267,8 @@ public virtual void RollbackTransaction()
         /// <summary>
         ///     Opens the connection to the database.
         /// </summary>
-        /// /// <param name="errorsExpected"> Indicate if the connection errors are expected and should be logged as debug message. </param>
+        /// ///
```
Comment (bricelam) [inline src/EFCore.Relational/Storage/RelationalConnection.cs]: Remove

File: src/EFCore.Sqlite.Core/Scaffolding/Internal/SqliteDatabaseModelFactory.cs [hunk 3]
```diff
@@ -95,9 +95,8 @@ public virtual DatabaseModel Create(DbConnection connection, IEnumerable<string>
                     }
                 }
 
-                var nullableKeyColumns = Enumerable
+                var nullableKeyColumns = databaseModel.Tables.Where(t => t.PrimaryKey != null).SelectMany(t => t.PrimaryKey.Columns)
                     .Concat(
-                        databaseModel.Tables.Where(t => t.PrimaryKey != null).SelectMany(t => t.PrimaryKey.Columns),
                         databaseModel.Tables.SelectMany(t => t.ForeignKeys).SelectMany(fk => fk.PrincipalColumns))
```
Comment (bricelam) [inline src/EFCore.Sqlite.Core/Scaffolding/Internal/SqliteDatabaseModelFactory.cs]: The old code is so much more readable.

File: src/EFCore/Metadata/Internal/Key.cs [hunk 4]
```diff
@@ -53,8 +54,7 @@ public Key([NotNull] IReadOnlyList<Property> properties, ConfigurationSource con
         /// </summary>
         public virtual EntityType DeclaringEntityType
         {
-            [DebuggerStepThrough]
-            get { return Properties[0].DeclaringEntityType; }
+            [DebuggerStepThrough] get { return Properties[0].DeclaringEntityType; }
```
Comment (bricelam) [inline src/EFCore/Metadata/Internal/Key.cs]: I like the old code better.

File: src/ef/CommandLineUtils/CommandArgument.cs [hunk 5]
```diff
@@ -15,14 +15,12 @@ public CommandArgument()
 
         public string Name { get; set; }
         public string Description { get; set; }
-        public List<string> Values { get; private set; }
+        public List<string> Values { get; }
```
Comment (bricelam) [inline src/ef/CommandLineUtils/CommandArgument.cs]: We shouldn't change anything under the `CommandLineUtils` directory. It's a fork and merging upstream changes will be more difficult.

File: src/ef/Commands/DatabaseDropCommand.cs [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.EntityFrameworkCore.Tools.Commands
 {
-    partial class DatabaseDropCommand
+    internal partial class DatabaseDropCommand
```
Comment (bricelam) [inline src/ef/Commands/DatabaseDropCommand.cs]: Not DRY. IMHO, only one partial should specify visibility.

File: src/EFCore/Metadata/Internal/Key.cs (hunk 4 above)
Comment (bricelam) [inline src/EFCore/Metadata/Internal/Key.cs]: Also, why no expression bodied getters and setters?

File: src/EFCore.Sqlite.Core/Scaffolding/Internal/SqliteDatabaseModelFactory.cs (hunk 3 above)
Comment (AndriySvyryd) [inline src/EFCore.Sqlite.Core/Scaffolding/Internal/SqliteDatabaseModelFactory.cs]: Agree, but that wasn't consistent with other places. Will reformat manually.

File: src/EFCore/Metadata/Internal/Key.cs (hunk 4 above)
Comment (AndriySvyryd) [inline src/EFCore/Metadata/Internal/Key.cs]: Would it be better with expression body?

File: src/ef/Commands/DatabaseDropCommand.cs (hunk 6 above)
Comment (AndriySvyryd) [inline src/ef/Commands/DatabaseDropCommand.cs]: R# limitation, will revert

File: src/EFCore/Metadata/Internal/Key.cs (hunk 4 above)
Comment (AndriySvyryd) [inline src/EFCore/Metadata/Internal/Key.cs]: We don't do this automatically because too many expression bodies in a file cause VS to become unresponsive

Comment (bricelam) [inline src/EFCore/Metadata/Internal/Key.cs]: Meh. Don't really care. If I ever happen to put an attribute on these I'll do it above, but I don't mind if they periodically get reformatted by R#.

**Pull request merged**